common/util/util.go

```diff
@@ -50,6 +50,12 @@ func MaxTime(a, b time.Time) time.Time {
 	return b
 }
 
+// NextAlignedTime returns the time after `t` that is aligned to an integer multiple of `align`
```

Comment:
returns the __earliest__ time after `t` that is ~aligned to~ an integer multiple of `align`